src/containers/AccountForm.jsx

```
@@ -1,34 +1,50 @@
+  name: yup.string("Enter your name").required("Name is required"),
+  email: yup.string("Enter your email").email("Enter a valid email").required("Email is required"),
+  password: yup.string("Enter your password").min(8, "Password should be of minimum 8 characters length").required("Password is required"),
 });
 
+const fieldSx = {
+  "& .MuiInputBase-input": {
+    fontSize: 15,
+  },
+};
+
 const AccountForm = ({ onSubmit }) => {
```

ESLint

[error] 'onSubmit' is missing in props validation

(react/prop-types)